src/frontend/sharedComponents/BottomSheetModal/Backdrop.tsx

```diff
@@ -9,25 +9,24 @@ export const Backdrop = ({
   style,
   ...rest
 }: BottomSheetBackdropProps) => {
-  const animatedOpacity = React.useMemo(
-    () =>
-      interpolate(animatedIndex, {
-        inputRange: [0, 1],
-        outputRange: [0, 0.3],
-        extrapolate: Extrapolate.CLAMP,
-      }),
-    [animatedIndex]
+  //Use memo was causing a flickering effect when the backdrop was opened.
+  const animatedOpacity = React.useRef(
+    interpolate(animatedIndex, {
+      inputRange: [0, 1],
+      outputRange: [0, 0.3],
+      extrapolate: Extrapolate.CLAMP,
+    })
   );
 
   const containerStyle = React.useMemo(
     () => [
       style,
       {
         backgroundColor: BLACK,
-        opacity: animatedOpacity,
+        opacity: animatedOpacity.current,
       },
     ],
-    [style, animatedOpacity]
+    [style, animatedOpacity.current]
```

Comment:
seems odd, but at least based on the example, i _think_ we want to keep this as the ref and not the current value of the ref since it's an animated value, which has weird rules and whatnot that i don't fully understand tbh. worth just qa-ing just change overall since it's pretty sensitive to such changes 😅

Reply:
a h wait i see what's happening. you converted that value to a ref from a memoized animated value. I _think_ that's okay but then i think we may need to update the `animatedOpacity` value if the `animatedIndex` updates, which is why i used the memo. does the flickering still happen if you add this effect and remove the ref from the deps array here?
 
 ```
 React.useEffect(() => {
   animatedOpacity.current = interpolate(animatedIndex, {
     // omitted for brevity
   })
 }, [animatedIndex])

Reply:
alternatively, maybe just simplifying to something closer to the example would work? i can't remember if the extraction was done due to visual bugs or something 😅 e.g. something along the lines of

```js
  const containerStyle = React.useMemo(
    () => [
      style,
      {
        backgroundColor: BLACK,
        opacity: interpolate(animatedIndex, {
          inputRange: [0, 1],
          outputRange: [0, 0.3],
          extrapolate: Extrapolate.CLAMP,
        }),
      },
    ],
    [style, animatedIndex]
  );
```

Reply:
i tried these, it did work. I dont want to spend too much time on this animation. so I am just turning it off and adding it to the backlog.

Reply:
issue resolved by https://github.com/digidem/mapeo-mobile/pull/946/commits/976099d03e442449227918da50300797f6abe459